src/main/java/co/empathy/academy/demo/DAOs/SearchDataAccessImp.java

```diff
@@ -108,6 +114,58 @@ public void analyzer(String index, String analyzer) throws Exception {
         esClient.indices().open(o -> o.index(index));
         
     }
+
+    @Override
+    public List<Movie> throwOrderByQuery(Query query, Map<String,Aggregation> aggs, String index, String key) throws ElasticsearchException, IOException {
+        SearchResponse<Movie> response = esClient.search(s -> s
+            .index(index)
+            .size(0)
+            .query(query)
+            .aggregations(aggs), Movie.class);
+
+        Aggregate Aggregate = response.aggregations().get(key);
+                
+        List<JsonData> result = Aggregate.topHits().hits().hits().stream()
+            .map(Hit::source)
+            .toList();
+
+        JSONArray jsonArray = new JSONArray(result.toString());
+        return JsonConversor.jsontoMovies(jsonArray); 
```

Comment:
Its nice to have a custom class, but please note that you can use jackson for your json to object conversions